roles/installer/templates/tower_service.yaml.j2

```diff
@@ -44,6 +44,8 @@ spec:
     app.kubernetes.io/component: awx
 {% if tower_ingress_type | lower == "loadbalancer" %}
   type: LoadBalancer
+{% elif tower_ingress_type | lower == "ingress" %}
+  type: ClusterIP
```

Comment:
If we review the code here, we have 3 options: `LoadBalancer`, `Route` and `Ingress`. However the `Route` option seems to be using the `ClusterIP` as well (see https://docs.openshift.com/enterprise/3.0/architecture/core_concepts/routes.html). So I believe we would need to actually add a new option called `NodePort` and then modify the logic like:

```yaml
{% if tower_ingress_type | lower == "loadbalancer" %}
  type: LoadBalancer
{% elif tower_ingress_type | lower == "nodeport" %}
  type: NodePort
{% elif  %}
  type: ClusterIP
{% endif %}

Reply:
Update the service template and docs to account for additional `tower_ingress_type: NodePort`